pyproject.toml

```diff
@@ -131,15 +131,18 @@ select = [
     "YTT",   # flake8-2020
 ]
 extend-ignore = [
-    'A002',    # builtin-argument-shadowing
-    'ANN401',  # any-type (mypy's `disallow_any_explicit` is better)
-    'E402',    # module-import-not-at-top-of-file (usually OS-specific)
-    'E501',    # line-too-long
-    'F403',    # undefined-local-with-import-star
-    'F405',    # undefined-local-with-import-star-usage
-    'PERF203', # try-except-in-loop (not always possible to refactor)
-    'PT012',   # multiple statements in pytest.raises block
-    'SIM117',  # multiple-with-statements (messes up lots of context-based stuff and looks bad)
+    "A002",    # builtin-argument-shadowing
+    "ANN401",  # any-type (mypy's `disallow_any_explicit` is better)
+    "E402",    # module-import-not-at-top-of-file (usually OS-specific)
+    "E501",    # line-too-long
+    "F403",    # undefined-local-with-import-star
+    "F405",    # undefined-local-with-import-star-usage
+    "PERF203", # try-except-in-loop (not always possible to refactor)
+    "PT012",   # multiple statements in pytest.raises block
+    "SIM117",  # multiple-with-statements (messes up lots of context-based stuff and looks bad)
+
+    # conflicts with formatter (ruff recommends these be disabled)
+    "COM812",
```

Comment:
Does Ruff have human-readable rule names? I thought it did... If so, I'd rather use those over some alphanumeric contextless IDs.

Reply:
I don't think it does

Reply:
ye they still can't be used ~anywhere